templates/vue/src/components/Lightbox/media/VideoMedia/KalturaMedia.vue

```diff
@@ -61,6 +61,7 @@ export default {
             currentTime,
           })
 
+          // kalturaVideo.sendNotification("changeVolume", 1)
```

Comment:
Perhaps this line can just be deleted instead of commented out.

Reply:
Oops! I will fix that.